homeassistant/components/knx/storage/config_store.py

```diff
@@ -33,6 +31,20 @@ class KNXConfigStoreModel(TypedDict):
     entities: KNXEntityStoreModel
 
 
+class PlatformControllerBase(ABC):
+    """Entity platform controller base class."""
+
+    @abstractmethod
+    async def add_entity(self, unique_id: str, config: dict[str, Any]) -> None:
```

Comment:
I'm thinking it would be even more clear to match the websocket API names here, `create_entity` and `update_entity`.